docs/guides/modules/deploy/pages/set-up-rollbacks.adoc

```diff
@@ -35,97 +35,88 @@ a| * Simple systems
 
-The following steps guide you through setting up a custom rollback pipeline. If you want details on how to *Rollback by workflow re-run*, see the <<rollback-by-workflow-re-run, Rollback by workflow re-run>> section at the end of this guide.
+=== Confirm GitHub App installation
 
-=== 2. Start the rollback setup
+NOTE: To set up rollback pipelines, you must connect your organization to the xref:integration:github-apps-integration[CircleCI GitHub App].
 
-. Select the btn:[Rollback] dropdown.
-. Select *Set up custom rollback pipeline* from the dropdown menu. This launches the rollback setup wizard.
+- If the GitHub App is not installed in your organization, the guided setup process prompts you to install it.
 +
-NOTE: To use the rollback by pipeline method, your organization must be connected to the *CircleCI GitHub App*. If the GitHub App is not installed in your org, the rollback setup process will automatically prompt you to install it during setup.
-
-The setup wizard guides you through configuring your custom rollback pipeline.
-
-image::guides:ROOT:deploy/set-up-a-rollback-pipeline.png[Set up a rollback pipeline]
+image::guides:ROOT:deploy/install-github-app.png[Install GitHub App]
 
-=== 3. Configure the rollback pipeline
-
-. From the "What repo are you deploying?" dropdown, select the repository you want to create rollbacks for.
-. Select btn:[Create pipeline definition] to proceed.
+- If the GitHub App is already installed in your organization, the **GitHub App Installation** step is automatically marked as completed.
++
+image::guides:ROOT:deploy/github-app-installed.png[GitHub App installed]
 
-CircleCI creates a pipeline definition called `rollback-pipeline` and uses the selected repository to store your rollback configuration.
 
-=== 4. Review the generated configuration
-After creating the pipeline definition, the modal will display a pre-generated configuration file for performing rollbacks.
+=== Add the deploy markers
 
-. Review the generated YAML configuration template to ensure you understand the rollback pipeline.
-. Select btn:[Commit Config] to create this rollback configuration template in your repository. The configuration will be committed to a new branch called `rollback-pipeline-setup` in your selected repository.
+. Expand the **Add deploy markers to existing pipeline** section.
+. In the **Environment Name** dropdown, select or type the xref:configure-deploy-markers#manage-environments[target environment].
+. In the **Component Name** dropdown, select or type the name of the component you want to rollback.
++
+image::guides:ROOT:deploy/add-deploy-markers-selection.png[Select deploy markers values]
 
-This config is a template rollback pipeline that includes the following:
+. The setup will auto-generate deploy marker commands based on your selections.
+. Copy the generated commands using the btn:[Copy] button and paste them into your existing config file where the deployment occurs.
```

Comment:
I think make it clear they are going to need to break this snipp up so the plan happens before their actual deployment and the update after? @rafaelmtorres just wondering if it would make sense to have two snippets not one for this reason?